contracts/core/AuraMinterOwner.sol

```diff
@@ -0,0 +1,135 @@
+    /// @dev Tracks cumulative minting to enforce MAX_TOTAL_CAP
+    uint256 public totalMinted;
+
+    /* -------------------------------------------------------------------
+       Events 
+    ------------------------------------------------------------------- */
+
+    /**
+     * @notice Emitted when AURA tokens are minted through this contract
+     * @param to The address that received the minted tokens
+     * @param amount The amount of tokens minted
+     * @param epoch The epoch in which the minting occurred
+     */
+    event AuraMinted(address indexed to, uint256 amount, uint256 epoch);
+
+    /* -------------------------------------------------------------------
+       Constructor 
+    ------------------------------------------------------------------- */
+
+    /**
+     * @notice Constructs the AuraMinterOwner contract
+     * @dev Initializes the contract with the AuraMinter address and transfers ownership to the DAO
+     * @param _auraMinter Address of the AuraMinter contract to wrap
+     * @param _dao Address of the DAO that will own this contract
+     */
+    constructor(address _auraMinter, address _dao) Ownable() {
+        auraMinter = AuraMinter(_auraMinter);
+        _transferOwnership(_dao);
+    }
+
+    /**
+     * @notice Calculates the current epoch number based on time elapsed since inflation protection ended
+     * @dev Epochs start counting from 1 when the AuraMinter's inflation protection time expires.
+     *      Each epoch lasts EPOCH_DURATION (52 weeks).
+     * @return The current epoch number (starting from 1)
+     */
+    function _getCurrentEpoch() internal view returns (uint256) {
+        if (block.timestamp <= auraMinter.inflationProtectionTime()) {
+            return 0;
+        }
+        return ((block.timestamp - auraMinter.inflationProtectionTime()) / EPOCH_DURATION) + 1;
+    }
+
+    /* -------------------------------------------------------------------
+       Core Functions 
+    ------------------------------------------------------------------- */
+
+    /**
+     * @notice Mints AURA tokens with epoch-based and total caps
+     * @dev Only the owner can mint. Enforces yearly epoch caps and total lifetime cap.
+     *      Minting is only allowed after the AuraMinter's inflation protection period ends.
+     * @param _to Address to receive the minted tokens
+     * @param _amount Amount of AURA tokens to mint (in wei, 18 decimals)
+     */
+    function mint(address _to, uint256 _amount) external onlyOwner {
+        require(_to != address(0), "Invalid recipient");
+        require(_amount > 0, "Zero amount");
+        require(block.timestamp >= auraMinter.inflationProtectionTime(), "Inflation protection active");
+
+        uint256 currentEpoch = _getCurrentEpoch();
+        uint256 currentEpochCap = currentEpoch * EPOCH_CAP;
+        uint256 totalAfterMinted = totalMinted + _amount;
+
+        require(totalAfterMinted <= currentEpochCap, "Exceeds epoch cap");
```

Comment:
This logic allows for carry over, so for example if I don't mint anything in the first year the second year I can mint 6M. The preferred logic is that inflation is capped per 52 week period. So instead we should make sure that only 3M can be minted per epoch.